applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -392,6 +392,25 @@ static void context_print(const struct shell *shell)
 	}
 }
 
+static void codec_qos_print(const struct shell *shell, struct bt_audio_codec_qos *qos)
+{
+	if (qos->phy == BT_AUDIO_CODEC_QOS_1M || qos->phy == BT_AUDIO_CODEC_QOS_2M) {
+		shell_print(shell, "\t\t\tPHY: %dM", qos->phy);
+	} else if (qos->phy == BT_AUDIO_CODEC_QOS_CODED) {
+		shell_print(shell, "\t\t\tPHY: LE Coded");
+	} else {
+		shell_print(shell, "\t\t\tPHY: Unknown");
+	}
+
+	shell_print(shell, "\t\t\tFraming: %s",
```

Comment:
Hmmmm, not sure if we can support framed from the perspective in application side.

Reply:
As long as it is supported in the controller it should be possible. This is not about we as a broadcast_sink can support, only what we can output in this tool. As far as the application goes we just push data down to the controller, and the controller will split it if necessary.